src/cluster/src/helm.rs

```diff
@@ -133,9 +133,18 @@ impl HelmClient {
 #[derive(Debug, Deserialize)]
 pub struct Chart {
     /// The chart name
-    pub name: String,
+    name: String,
     /// The chart version
-    pub version: String,
+    version: String,
+}
+
+impl Chart {
+    pub fn version(&self) -> String {
```

Comment:
What if this returned `&str` so we don't have to copy? It'd look like
```rust
pub fn version(&self) -> &str {
    &self.version
}
```
The same could go for the `name` function below